adserver/templates/adserver/reports/publisher_embed.html

```diff
@@ -0,0 +1,49 @@
+{% extends "adserver/reports/base.html" %}
+{% load humanize %}
+{% load i18n %}
+
+
+{% block title %}{% trans 'Publisher Embed' %} - {{ publisher }}{% endblock %}
+
+
+{% block heading %}{% blocktrans %}Publisher Embed for {{ publisher }}{% endblocktrans %}{% endblock heading %}
+
+{% block filters %}
+{% endblock %}
+
+{% block breadcrumbs %}
+  {{ block.super }}
+  <li class="breadcrumb-item active">{{ publisher }}</li>
+{% endblock breadcrumbs %}
+
+
+{% block summary %}
+  <section>
+    <div>
+      <h2></h2>
+      <div class="row col-md-8">
+
+        <p>
+          This code will allow you to get started with showing ads on your site.
+        </p>
+
+        <pre>
```

Comment:
You might get something a little better if you do `<pre><code>...</code></pre>`: https://getbootstrap.com/docs/4.1/content/code/